guides/adr/0012-formalize-accredited-provider-relationships.md

```diff
@@ -0,0 +1,380 @@
+# 12. Formalize Accredited Provider relationships
+
+Date: 23/10/24
+
+## Status
+
+Proposal
+
+## Context
+
+
+### Evolving provider relationships
+The structural and conceptual relationships among and between providers and users has changed in many ways since this codebase was started.[[1]]
+
+Providers have always had a relationship between one another in the form of course accreditation. More recently, this relationship has been extended to delegating this accreditation.
+
+We have Accredited provider relationships that exist solely for organisational reasons. This happens when: 
+
+
+| Provider | Is accredited? | Is accredited by |
+| ---      | ---            | ---              |
+| A        | Yes            | -                |
+| B        | Yes            | A                |
+| C        | No             | B                |
+
+
+| Provider | Is accredited? | "Is accredited by" | asdf | Who ratifies their courses? |
```

Comment:
`Is accredited by` is incorrect since this is what DfE does. It's more accurate to say `Is ratified by`.

Reply:
This is describing a direct relationship between two providers, not the relationship through a course.